.env.base

```diff
@@ -36,7 +36,7 @@ REACT_APP_FEATURE_THORCHAIN_LP_WITHDRAW=true
 REACT_APP_FEATURE_ACCOUNT_MANAGEMENT=true
 REACT_APP_FEATURE_ACCOUNT_MANAGEMENT_LEDGER=true
 REACT_APP_FEATURE_RFOX=true
-REACT_APP_FEATURE_RFOX_DASHBOARD=false
```

Comment:
I think we're leaving it toggled off for now and letting marketing toggle on manually

Reply:
I'm not sure that's true - the base feature, `REACT_APP_FEATURE_RFOX`, is already on.

The dashboard is only off because it was showing placeholder values.

Reply:
ah okay 🧠